docs/changelog/118968.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 118968
```

Comment:
Let's keep this changelog here? And merge the summary into this changelog entry.